modules/search/client/components/search.api.js

```diff
@@ -0,0 +1,10 @@
+export async function searchUsers(query) {
+  const response = await fetch(`/api/users?search=${query}`, {
```

Comment:
We can now rebase this PR and switch to use `axios` instead of `fetch`, see docs:

https://github.com/Trustroots/trustroots/blob/master/docs/React.md#api-calls-from-react

Reply:
Is this service already anywhere or is it going to be anywhere anytime soon?

Reply:
I don't think was added yet. Axios library is in already tho.

Reply:
Adding this in https://github.com/Trustroots/trustroots/pull/984